### ISSUE TITLE
add Grails 3 support

### ISSUE THREAD

**Issue by meltzow** [peh/react-grails-asset-pipeline#5]
any grails3 support planned?

**Comment by peh** [comment]
Yes. But i can not give you any estimates. You could try adding https://github.com/peh/react-asset-pipeline to your dependencies (it is on bintray). As this Plugin is doing nothing else.

**Comment by meltzow** [comment]
Hi,
thanks for fast answer. 
I've added runtime

```
maven {
    url  "http://dl.bintray.com/errbuddy/plugins"
}
```

and

```
 'net.errbuddy.plugins:react-asset-pipeline:1.0.8'
```

 to my build.gradle. and add an *.jsx file to my grails-app/assets/javascript folder but it looks, that nothing happens. The asset is not inserted. Any ideas why?

**Comment by peh** [comment]
Mh. Nothing concrete sorry. Will have to look into it but i will not have any real pc Access till friday as i'm in Barcelona right now. Will have a Look as soon as i'm back.

**Comment by vmptk** [comment]
Hi, 

Will there be any support for grails 3 ?

**Comment by peh** [comment]
hey,

short answer: yes. but i still don't know when. currently i'm a bit short on time.

**Comment by rogerschlachter** [comment]
Is the Grails 3 support still incoming?

**Comment by rogerschlachter** [comment]
For what it's worth, I was able to check out the grails 3 branch here, add a missing import for the GrailsApplication class on the `react-grails-asset-pipeline/grails-app/taglib/reactassetpipeline/ReactTagLib.groovy` then install it locally and use it in another project. My .jsx file is getting picked up and things are working fine.

**Comment by peh** [comment]
i highly recommend using 
https://github.com/errbuddy/babel-asset-pipeline for that in grails3
Babel is comming with build in JSX transformation and this way you are not dependant on any provided react version.
to use it you could just add

```
compile 'net.errbuddy.plugins:babel-asset-pipeline:1.3.0'
```

to your dependencies in build.gradle and it works
then you can download react from http://facebook.github.io/react/downloads.html and you are good to go
